_stbt/stbt-power.sh

```diff
@@ -0,0 +1,194 @@
+#!/bin/bash
+
+# WARNING: DON'T ADD SUPPORT FOR NEW POWER DEVICES TO THIS SCRIPT.
+# WE WANT TO PORT IT TO PYTHON FIRST. SEE
+# https://groups.google.com/d/msg/stb-tester/0wFNSIimHn0/317YMmc4CesJ
```

Comment:
You could update this message to point the developer to the python script instead.